forwarder/forwarder.go

```diff
@@ -0,0 +1,57 @@
+package forwarder
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/Financial-Times/transactionid-utils-go"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type Forwarder interface {
+	Forward(tid string, uuid string, collectionType string, reqBody []byte) (*ForwarderResponse, error)
```

Comment:
I make everything a pointer because of two things:
1) I'm never sure what happens in go, if something will gets needlessly copied or if the internals are actually a pointer so there no performance impact due to this. In our case, the response body might be large and I wanted to be sure that it's not getting copied.
2) I wanted to return `nil, error` in case something bad takes place. Call it a java carryover

If you want, I can change the response to be "non pointer". If I do this, I'll change most all other responses, for consistency.